backend/app/crud/prompt_version.py

```
@@ -0,0 +1,273 @@
+        prompt_id=prompt_id,
+        project_id=project_id,
+    )
+    if not prompt:
+        logger.error(
+            f"[get_prompt_versions] Prompt not found | Prompt ID: {prompt_id}, Project ID: {project_id}"
+        )
+        raise HTTPException(status_code=404, detail="Prompt not found")
+
+    stmt = (
+        select(PromptVersion)
+        .where(PromptVersion.prompt_id == prompt_id, PromptVersion.is_deleted == False)
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not PromptVersion.is_deleted:` for false checks

Replace with `not PromptVersion.is_deleted`

(E712)